cmake/tpls/DyninstBoost.cmake

```diff
@@ -17,33 +17,27 @@
 
 include_guard(GLOBAL)
 
-# Need at least 1.71 for a usable BoostConfig.cmake
-set(_min_version 1.71.0)
```

Comment:
We use this to synchronize the version requirements with the containers (see [docker/Dockerfile.ubuntu](https://github.com/dyninst/dyninst/blob/master/docker/Dockerfile.ubuntu).

Reply:
Whoops, fixed.